lua/lvim/core/telescope/custom-finders.lua

```diff
@@ -14,6 +14,7 @@ local utils = require "lvim.utils"
 function M.find_lunarvim_files(opts)
   opts = opts or {}
```

Comment:
The problem is that we're merging over the theme.

Reply:
ah i see, i wanted to make it like this
<img width="1850" alt="Screen Shot 2022-02-27 at 3 39 52 PM" src="https://user-images.githubusercontent.com/10992695/155881795-ed171799-cc16-4f4f-b6e4-ef8030064662.png">
<img width="1912" alt="Screen Shot 2022-02-27 at 3 40 04 PM" src="https://user-images.githubusercontent.com/10992695/155881796-621d4a5d-4f2d-43a4-9f88-599871db945a.png">

i'll revert the changes and do a separate one for custom styles

Reply:
Yeah, creating a custom style/theme should do it. It's tricky when modifying the global default.